packages/orbit-components/src/Modal/ModalSection/index.tsx

```diff
@@ -43,7 +43,7 @@ export const ModalSectionWrapper = ({
             ],
         !isMobileFullPage && [
           "first-of-type:rounded-t-modal lm:first-of-type:rounded-t-modal",
-          "last-of-type:rounded-b-modal lm:last-of-type:rounded-t-modal lm:[&_~_.orbit-modal-footer]:last-of-type:mt-0",
```

Comment:
Just one question: we removed `rounded-t-modal` and replaced it with `rounded-b-modal`; was the issue all along that we just didn't have the correct CSS class?

Looking at the previous state, it doesn't seem to make sense and was probably a copy-paste-forgot to update the class kind of situation.

Reply:
Actually, the top border-radius is also needed when there's no header. I'll adjust it and add visual tests.

Reply:
This has been done.